paclab/settings.py

```diff
@@ -125,4 +125,12 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
\ No newline at end of file
+MEDIA_URL = '/media/'
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
we are using gmail's smtp? is that even allowed?

Reply:
I have tested the password reset and it appears to be working correctly with gmail. 
There is a way to set-up an email backend with just the server I believe though if this isn't a good option.

Reply:
Right but the point is you had to give your gmail user/password.  That is not something we want sitting in a file somewhere...